lumicks/pylake/force_calibration/tests/test_touchdown.py

```diff
@@ -148,14 +148,20 @@ def test_exp_sine_fits(decay, amplitude, frequency, phase_shift):
 @pytest.mark.filterwarnings("ignore:Covariance of the parameters could not be estimated")
 @pytest.mark.filterwarnings("ignore:Surface detection failed")
 def test_insufficient_data(mack_parameters):
-    stage_positions, simulation = simulate_touchdown(102.5, 103.5, 0.01, mack_parameters)
+    stage_positions, simulation = simulate_touchdown(102.5, 104, 0.01, mack_parameters)
```

Comment:
Is it really called `mack_parameters`?

Reply:
[Yes](https://pmc.ncbi.nlm.nih.gov/articles/PMC3482253/). :D

Reply:
Gotcha, thanks for the `vanlier_comment`.